packages/react/src/storefront-client.ts

```diff
@@ -118,6 +123,16 @@ type OverrideTokenHeaderProps = Partial<
 >;
 
 type StorefrontClientReturn = {
+  /**
+   * Creates the fully-qualified URL to your myshopify.com domain.
+   *
+   * By default, it will use the config you passed in when calling `createStorefrontClient()`. However, you can override the following settings on each invocation of `getStorefrontApiUrl({...})`:
```

Comment:
```suggestion
   * By default, it will use the config you passed in when calling `createStorefrontClient()`. However, you can override the following settings on each invocation of `getShopifyDomain({...})`:
```